src/main/java/io/jenkins/plugins/analysis/core/model/FilesScanner.java

```diff
@@ -31,23 +33,25 @@
     private final String filePattern;
     private final IssueParser parser;
     private final String encoding;
+    private final boolean followSymlinks;
 
     /**
      * Creates a new instance of {@link FilesScanner}.
-     *
-     * @param filePattern
+     *  @param filePattern
      *         ant file-set pattern to scan for files to parse
      * @param tool
      *         the static code analysis tool that reports the issues
      * @param encoding
-     *         encoding of the files to parse
```

Comment:
This is a IntelliJ bug, removes a line of the comment.

Please reformat the comment after re-adding the line.